orchagent/aclorch.cpp

```diff
@@ -2390,10 +2398,14 @@ void AclOrch::queryAclActionCapability()
         SWSS_LOG_WARN("Failed to query maximum ACL action count - "
                 "API assumed to be not implemented, using defaults capabilities for both %s and %s",
                 STAGE_INGRESS, STAGE_EGRESS);
-        for (auto stage: {ACL_STAGE_INGRESS, ACL_STAGE_EGRESS})
+        task_process_status handle_status = handleSaiGetStatus(SAI_API_SWITCH, status);
```

Comment:
>status [](http://example.com/codeflow?start=79&length=6)

The same

Reply:
Removed the handling for the ones with default values in failures.